hugo/layouts/report.html

```diff
@@ -382,6 +382,24 @@ <h3 class="mb-3">Top Characteristics</h3>
 
   {{ end }}
 
+  {{/* Renders a field label with an info tooltip if a description exists */}}
+  {{ define "label-with-tooltip" }}
+    {{- $code := .code -}}
+    {{- $info := index .Params.var_info $code -}}
+    {{- $label := cond (and $info (ne $info.label "")) $info.label $code -}}
+    {{- $desc  := cond (and $info (ne $info.description "")) $info.description "" -}}
+
+    {{- if $desc }}
+      <span class="d-inline-block" tabindex="0" data-bs-toggle="tooltip" data-bs-title="{{ $desc }}">
+        <span style="text-decoration: underline #294298; text-decoration-style: dotted; text-decoration-thickness: 2px">
+          {{ $label }}
+        </span>
+      </span>
+    {{- else -}}
+      {{ $label }}
+    {{- end -}}
+  {{ end }}
+
```

Comment:
This template packages both the human readable label and the tooltip if it exists.